wolfcrypt/src/ecc.c

```diff
@@ -6931,6 +6931,14 @@ int wc_ecc_import_point_der_ex(byte* in, word32 inLen, const int curve_idx,
                     (ECC_CURVE_FIELD_PRIME | ECC_CURVE_FIELD_AF |
                         ECC_CURVE_FIELD_BF));
 
+            /* validate prime is prime */
```

Comment:
This seems like a big performance hit. Shouldn't this only be used if its a custom curve provided and not one of the static ecc_sets? That would be `curve->id == ECC_CURVE_CUSTOM`